sstables/sstables_manager_subscription.hh

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2025-present ScyllaDB
+ *
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#pragma once
+
+#include <boost/signals2/dummy_mutex.hpp>
+#include <boost/signals2/signal_type.hpp>
+
+#include "sstables/generation_type.hh"
+
+namespace bs2 = boost::signals2;
+
+namespace sstables {
+
+enum class manager_event_type {
```

Comment:
I think the names should be more verbose, or, in past tense, as these are events being sent after the event happened.

Reply:
past tense is good

Reply:
changed to past tense in v5 ([a8bc344](https://github.com/scylladb/scylladb/commit/a8bc344156a32b48877ac1fa90736f07f9d77a6c))